src/components/FileUploaderDetails/UploaderProgressDetails.part.tsx

```diff
@@ -15,7 +15,7 @@ const StyledModal = styled(Modal)`
   max-width: 600px;
 `;
 
-export interface UploaderProgressDetailsProps extends UploadProgressDetailsLabels {
+export interface UploaderProgressDetailsProps extends UploadProgressDetailsLabels, Translate {
```

Comment:
Translate can be a part of `StandartProps`

Reply:
You mean these?

```ts
export interface StandardProps {
  /**
   * Places the given class on the element.
   */
  className?: string;
  /**
   * An optional theme which can be passed down to a component.
   */
  theme?: PreciseTheme;
  /**
   * The style prop for explicitly overriding some CSS styles.
   */
  style?: React.CSSProperties;
}
```

I would not place `translate` here. Most of our components do not use this prop. `StandardProps` is an interface that makes sense for *all* components (e.g., every component consumes `style`, `className`, ... in one way or the other).

Reply:
ok, agreed